examples/introduction.ipynb

```diff
@@ -9,7 +9,28 @@
   },
   {
    "cell_type": "code",
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
    "execution_count": 88,
+=======
+=======
+>>>>>>> df872e6 (Added vegalite categorial and quantitive chart lists)
+<<<<<<< HEAD
+   "execution_count": 10,
+=======
+   "execution_count": 22,
+>>>>>>> 5f66182 (Implemented mark customization)
+<<<<<<< HEAD
+>>>>>>> 0893756 (Implemented mark customization)
+=======
+=======
+   "execution_count": 1,
+>>>>>>> d60a3bf (Added vegalite categorial and quantitive chart lists)
+>>>>>>> df872e6 (Added vegalite categorial and quantitive chart lists)
+=======
+   "execution_count": 10,
+>>>>>>> 52d8dd6 (Implemented mark customization subtab)
```

Comment:
You might have accidentally pushed the notebook with the merge conflict annotations in them. Could you replace the notebook before merging?